packages/mdc-select/mdc-select.scss

```diff
@@ -21,7 +21,6 @@
 @import "@material/ripple/common";
 @import "@material/ripple/mixins";
 @import "@material/rtl/mixins";
-@import "@material/list/mixins";
```

Comment:
This snuck into this PR I think

Reply:
I deleted this, because I accidentally added it in one of my previous commits.